src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(
+                new Bulldozer(),
+                new Bulldozer(),
+                new Bulldozer()

```

Comment:
use parameterized constructors. at the moment, you've added fields to this class, but not initialized them